### PR TITLE
handle edge cases with a and an

### PR DESCRIPTION
**Pull request by lsmith77**
https://wittyworks.slack.com/archives/C0269P5AUKZ/p1677939480842919
image

https://wittyworks.slack.com/archives/C0269P5AUKZ/p1677934310544589
image

fixes edge cases in c4d042d6e860babd0fc60f14a7c9827b91335dd4cc3afe78f3d3f3b4cf3aa2e7

### CODE REVIEW

File: tests/test_grammatically_correct_alternatives/test_english_alternative_context/output.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
                 }
             ],
             "category": "gendered",
-            "context": "We hanged them up. We analyzed if this works. Are you a chairman or an addict?",
+            "context": "We hanged them up. We analyzed if this works. Are you a chairman or an addict? Are you a top-performer? This is an eye-opener.",
```
Comment (lsmith77) [inline tests/test_grammatically_correct_alternatives/test_english_alternative_context/output.json:24]: @tje-ww does the below look correct now?

Comment (tje-ww) [reply tests/test_grammatically_correct_alternatives/test_english_alternative_context/output.json:24]: Line 96: "substance use" isn't a valid alternative in this context. Should I move it to "inspiration?"

Comment (lsmith77) [reply tests/test_grammatically_correct_alternatives/test_english_alternative_context/output.json:24]: yes please

**Review by lnazarenko — APPROVED** [review]
